pep-0001.txt

```diff
@@ -738,7 +740,7 @@ Once the PEP is ready for the repository, a PEP editor will:
   status as "Draft".
 
 * Add the PEP to a local fork of the PEP repository.  For workflow
-  instructions, follow `The Python Developers Guide <http://docs.python.org/devguide>`_
+  instructions, follow `The Python Developers Guide <https://devguide.python.org/>`_
 
   The git repo for the peps is::
```

Comment:
Given you've updated the other links, no reason for it to still use a verbatim block that the user has to manually copy, paste and navigate to in their browser instead of simply a link. I suggest just deleting this block and making the change above instead to link it

Reply:
Thanks, like this? cb1fdfec028db2fe0cf16552216f743863e56cbf

Reply:
Sorry, I meant just deleting the whole block (couldn't make it a suggestion as it was outside the scope GitHub would let me), since the PEP repo is already referred to and directly linked two lines above, so restating that is now redundant.